luarules/gadgets/dbg_test_env_helper.lua

```diff
@@ -0,0 +1,78 @@
+function gadget:GetInfo()
+	return {
+		name = "Test Environment Helper",
+		desc = "Helper to setup test environment conditions",
+		license = "GNU GPL, v2 or later",
+		layer = 9999,
```

Comment:
why 9999?

Reply:
Just a copy paste artifact... I think it doesn't matter.